scripts/manpage_generation.sh

```diff
@@ -5,15 +5,41 @@ MANPAGE_DIR="man"
 
 echo Generating manpages
 
-VER=$(grep -oP '^Version:\s+\K\S+' packaging/convert2rhel.spec)
+# Detect changes in __init__.py
+if git diff --quiet HEAD~1 -- convert2rhel/__init__.py; then
+    # No changes detected, exit with 0
+    echo "No new version detected"
+    exit 0
+else
+    echo "enter changes deteced"
+    # Changes detected, generate manpages
```

Comment:
Good solution

```suggestion
# Detect changes in __init__.py
if git diff --quiet HEAD~1 -- convert2rhel/__init__.py; then
    # No changes detected, exit with 0
    echo "No new version detected"
    exit 0
else
    echo "New version detected"
    # Changes detected, generate manpages
```

Reply:
That was a template for when I was working on this. I can get that changed!